### PR TITLE
test suite of Importing sample data to dashboard 

### LINKED ISSUE #13: Add sample data “Sample eCommerce orders”

### PR DESCRIPTION
**Pull request by Tengda-He**
### Description
test suite of Importing sample data to dashboard 
 
### Issues Resolved
#13 
 
### Test
https://user-images.githubusercontent.com/89666799/142688978-954e0b20-b177-4c22-9097-ba55ac214bfc.mp4

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
   "reporterOptions": {
     "configFile": "reporter-config.json"
   },
+  "viewportWidth": 2000,
```
Comment (seraphjiang) [inline cypress.json:8]: kind of beyond typical resolution 1920x  1080
any reason, we have to set this width to pass the test cases?

File: cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            miscUtils.visitPage('app/home#')
+        })
+
+        after(() => {
+
+        })
+
+        it('checking opensearch_dashboards_overview display', () => {
+            // Check that opensearch_dashboards_overview is visable
+            commonUI.checkElementExists('a[href="/_dashboards/app/opensearch_dashboards_overview"]', 1)
```
Comment (seraphjiang) [inline cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js:35]: not all dashboards instance will set `server.basePath: /_dashboards`
could we extract the `_dashboards` to basePath parameter as part of test run parameter?

File: cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            miscUtils.addSampleData()
+        })
+
+        after(() => {
+            miscUtils.removeSampleData()
+        })
+
+        it('checking ecommerce dashboards displayed', () => {
+            miscUtils.viewData('ecommerce')
+            commonUI.checkElementContainsValue('span[title="\[eCommerce\] Revenue Dashboard"]', 1, '\\[eCommerce\\] Revenue Dashboard')
```
Comment (seraphjiang) [inline cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js:86]: move ``div[data-test-subj="markdownBody"] > h3`` to page model library or at least make it constants for readability and maintainability 

e.g.
```
const dashboards_homepage_xxx_dashboards_yyy_label_element_xpath
```

Comment (Tengda-He) [reply cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js:86]: will change this

File: cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js (hunk 2 above)
Comment (Tengda-He) [reply cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js:35]: will make the change to address this.

File: cypress.json (hunk 1 above)
Comment (Tengda-He) [reply cypress.json:8]: the default view point dimension is not large enough, causing the notification div cover the button, so we double the default size, to allow the notification div display without overlap other UI elements.

Comment (seraphjiang) [reply cypress.json:8]: Not too many customers will have ulta wide screen. Could we file a ux bug to notification team

Comment (Tengda-He) [reply cypress.json:8]: the error notification only shows on Cypress bundled chrome but not the normal chrome browser, so I was wondering something with Cypress instead.

**Review by seraphjiang — APPROVED** [review]

**Review by seraphjiang — APPROVED** [review]

**Comment by tianleh** [comment]
A good starting point. We may need to think about the test structure and its name a bit. 
Currently it is `cypress/integration/vanilla-opensearch-dashboards/dashboard_sample_data_spec.js`

1. name
It is actually a simple release sanity test case. The name does not seem easily reveal this.
2. dir
Since this test case repo will be for bundle OSD, thus `vanilla-opensearch-dashboards` will later deprecated. Once the plugin team onboards this, they will be put under 

`cypress/integration/AD` which will have two sub folders for security enabled and security disabled. Thus eventually this test case will be similarly under `cypress/integration/dashboards/security-disabled/`

**Review by tianleh — APPROVED** [review]

**Pull request merged**